src/coreimage.cs

```diff
@@ -1346,6 +1382,16 @@ interface CIImage : NSSecureCoding, NSCopying {
 		[Wrap ("FromCGImage (image, options == null ? null : options.Dictionary)")]
 		CIImage FromCGImage (CGImage image, [NullAllowed] CIImageInitializationOptionsWithMetadata options);
 
+		[iOS (13,0)][TV (13,0)][Watch (6,0)][Mac (10,15)]
+		[Static]
+		[Export ("imageWithCGImageSource:index:options:")]
```

Comment:
```suggestion
		[EditorBrowsable (EditorBrowsableState.Advanced)]
		[Export ("imageWithCGImageSource:index:options:")]
```

I think you want to encourage people to use the strongly typed version.